README.md

```diff
@@ -1,30 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+> First project setting up a "Hello world" repository
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTLM
```

Comment:
- [ ] OPTIONAL Hi, I found a typo in your READEME, please fix this. Paying attention to the detail in your README is crucial as this is what people will look at when viewing your repository.  👍